plan/config.go

```diff
@@ -57,12 +59,19 @@ func ReadConfigFromEnviron() (*Config, error) {
 			behaviors[behavior.Name] = behavior
 		}
 	}
+
+	jsonReportPath := os.Getenv(jsonReportPathKey)
+	if jsonReportPath == "" {
+		jsonReportPath = "/crossdock/report.json"
```

Comment:
I’ll follow up with that change with the coming refactor to reporters so they don’t expose channels in their interface.